### PR TITLE
Changed e2e basic test flavor

### PR DESCRIPTION
**Pull request by huutomerkki**
Now matches that of normal e2e integration tests.

### CODE REVIEW

File: jenkins/scripts/integration_test.sh [hunk 1]
```diff
@@ -68,7 +68,7 @@ if [[ "${TESTS_FOR}" == "feature_tests"* ]] || [[ "${TESTS_FOR}" == "e2e_tests"*
 fi
 echo "Running in region: ${OS_REGION_NAME}"
 
-if [[ "${TESTS_FOR}" == "feature_tests"* || "${TESTS_FOR}" == "e2e_tests"* && "${GINKGO_FOCUS}" != "integration" ]]; then
+if [[ "${TESTS_FOR}" == "feature_tests"* || "${TESTS_FOR}" == "e2e_tests"* && ( "${GINKGO_FOCUS}" != "integration" && "${GINKGO_FOCUS}" != "basic" ) ]]; then
```
Comment (kashifest) [inline jenkins/scripts/integration_test.sh:71]: Do we have feature_tests* anymore ?

Comment (mboukhalfa) [reply jenkins/scripts/integration_test.sh:71]: True We have to clean that but I guess we can do it in a separate PR it is not related to adding the basic integration test

**Review by kashifest — APPROVED** [review]

**Pull request merged**